packages/fx-core/src/common/localEnvMultiProvider.ts

```diff
@@ -6,7 +6,7 @@ import * as dotenv from "dotenv";
 import * as fs from "fs-extra";
```

Comment:
Just call it `LocalEnvProvider`.
Move it to *common/local/*

Reply:
Got it.

Reply:
Update class name as well.